composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/InstalledTestDescriptorModel.kt

```diff
@@ -53,6 +58,21 @@ data class InstalledTestDescriptorModel(
 
     val isExpired get() = expirationDate != null && expirationDate < LocalDateTime.now()
 
+    val isDefaultTestDescriptor get() = id.value in 10470..10474 // TODO(aanorbel): switch to OONI reserved namespace
+
+    val key get() = if (isDefaultTestDescriptor) {
```

Comment:
This is more like a settings key now no? Just to make it clear why we still need this. Because for all other purposes now we can use the ID.